lib/node_modules/@stdlib/math/base/special/minabsf/docs/repl.txt

```diff
@@ -0,0 +1,39 @@
+
+{{alias}}( x, y )
+    Returns the minimum absolute absolute single-precision
+    floating-point number.
+
+    If any argument is `NaN`, the function returns `NaN`.
+
+    When an empty set is considered a subset of the extended reals (all real
+    numbers, including positive and negative infinity), positive infinity is the
+    greatest upper bound. Similar to zero being the identity element for the sum
+    of an empty set and to one being the identity element for the product of an
+    empty set, positive infinity is the identity element for the minimum, and
+    thus, if not provided any arguments, the function returns positive infinity.
```

Comment:
The parameters are mandatory per the types and I am getting `NaN` back when calling it without any arguments. 

```suggestion
```

The note makes sense for  `math/base/special/maxn` and `math/base/special/minn`, which have a variadic number of parameters, but should be removed here.

Reply:
Thanks for explaining!